src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,270 @@
+        self._initOutputDirectory()
+
+        if self.timeoutSec > 0:
+            # Run with timeout:
+            with self.__timeout(self.timeoutSec):
+                try:
+                    self._run()
+                except TestFailedError:
+                    printTester(f"'{self.name}' failed.")
+                except TimeoutError:
+                    self._timeout()
+                except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)